### ISSUE TITLE
Configurar Docker

### ISSUE THREAD

**Issue by Joabesv** [Joabesv/ufabc-next-ms-users#11]
Implement Docker-compose file for a mongodb instance

Some notes

**DO NOT** use latest, MONGODBv6 introduced some **BIG** breaking changes, that need to be revisited later
**USE ENV** Docker-ComposeV3.8, is already as safe as it can be to use .env files to keep your secrets away from the file,
here's a little example
```bash
# Crucial part
version: '3.8'
# other configs you want to put...
env_file:
 - .env
```
pretty simple right?
### In the .env you should add these keys, they came from the mongo docker docs, read more here
https://hub.docker.com/_/mongo // Enviroment Variables section
```
 MONGO_INITDB_ROOT_USERNAME: root
 MONGO_INITDB_ROOT_PASSWORD: example
```

**Comment by ViniciussSantos** [comment]
I can work on this one. Just mongodb ? Maybe we could add the redis instance to the docker-compose file or even a dockerfile config.

**Comment by Joabesv** [comment]
For the initial steps, and to invite more collaborators in future releases, a simple Mongo docker is enough! You can open issues about the Redis with Dockerfile config though, feel free to do it man :)

**Comment by Joabesv** [comment]
closed by #18
